views/scss/qti/_qti-base.scss

```diff
@@ -217,6 +217,9 @@ $mainContainer : '.qti-item';
 
     .qti-block {
         border:1px solid transparent;
+        & img {
+            width: inherit !important;
```

Comment:
And this doesn't fix the issue with the relative size images, for which a `.label-content`, containing a `.qti-block` should have a width of `100%`

Reply:
With inherit, he will use the value of their parent. I use a circle as responsive with for example 6% and square with a fixed value of ~100px and it works for me
![imagen](https://user-images.githubusercontent.com/34692207/78803346-2e41cc80-79bf-11ea-904b-005c47eb7425.png)

Reply:
No, it doesn't. The relative size is being set to the `img` itself via `width` attribute. Your so-called relative `6%` is in fact just `width: inherit` and nothing more.
![image](https://user-images.githubusercontent.com/2943256/78866270-5e30b480-7a3f-11ea-9a29-f81d869f1102.png)

Reply:
No, it doesn't. The relative size is being set to the `img` itself via `width` attribute. Your so-called relative `6%` is in fact just `width: inherit` and nothing more.
![image](https://user-images.githubusercontent.com/2943256/78866270-5e30b480-7a3f-11ea-9a29-f81d869f1102.png)